<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Descriptive Analysis</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/table1-1.0/table1_defaults.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">SPARE-AD</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="descriptive_analysis.html">Descriptive Analysis</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Descriptive Analysis</h1>

</div>


<pre class="css"><code>.bggreen {
background-color: rgb(206, 237, 202);
}</code></pre>
<style type="text/css">
.bggreen {
background-color: rgb(206, 237, 202);
}
</style>
<div id="process-raw-hamarnized-dataset" class="section level1">
<h1>Process Raw Hamarnized Dataset</h1>
<p>PHC_Visit frequency table:</p>
<pre><code>## 
##    1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
## 5884 2531 2459 1756 1299 1003  792  511  421  243  198  142  107   70   40   34 
##   17   18   19   20   21   22   23   24   26 &lt;NA&gt; 
##   21   16    6    6    3    3    1    2    1 5283</code></pre>
<p>Get number of individuals</p>
<pre class="r"><code>print(length(unique(raw$Phenotype_ID)))</code></pre>
<pre><code>## [1] 10181</code></pre>
<p>Subset and get baseline data</p>
<pre class="r"><code>## detect individuals with same age at different PHC visit
dupAge &lt;- raw %&gt;%
  select(Phenotype_ID,PHC_Age_T1) %&gt;%
  group_by(Phenotype_ID,PHC_Age_T1) %&gt;%
  filter(n()&gt;1)

print(&quot;Inviduals with duplicate PHC_Age_T1&quot;)</code></pre>
<pre><code>## [1] &quot;Inviduals with duplicate PHC_Age_T1&quot;</code></pre>
<pre class="r"><code>print(raw[raw$Phenotype_ID %in% dupAge$Phenotype_ID,])</code></pre>
<pre><code>##        Study Phenotype_ID Study_Visit PHC_SCANDATE PHC_Visit PHC_Sex
## 14409   NACC   NACC559323           1         &lt;NA&gt;        NA       1
## 14410   NACC   NACC559323           5         &lt;NA&gt;        NA       1
## 14411   NACC   NACC559323           7         &lt;NA&gt;         7       1
## 20114 WHICAP       W62307        &lt;NA&gt;   2012-11-14        NA       1
## 20115 WHICAP       W62307        &lt;NA&gt;   2012-11-14        NA       1
## 20116 WHICAP       W62307        &lt;NA&gt;   2018-01-31        NA       1
## 20427 WHICAP       W82050        &lt;NA&gt;   2011-05-11         1       2
## 20428 WHICAP       W82050        &lt;NA&gt;   2011-05-11         1       2
##       PHC_Education PHC_Ethnicity PHC_Race PHC_Age_T1 PHC_Diagnosis
## 14409            16             2        5   71.58385             1
## 14410            16             2        5   71.58385             1
## 14411            16             2        5   73.58248             1
## 20114            14             2        3   84.87064             1
## 20115            14             2        3   84.87064             1
## 20116            14             2        3   90.08350             1
## 20427            12             1        6   69.35524             1
## 20428            12             1        6   69.35524             1
##       PHC_ScannerBatchName   SPARE_AD SPARE_BA
## 14409          SIEMENS_3.0 -0.5219044 52.99583
## 14410          SIEMENS_3.0 -1.6240788 41.40565
## 14411          SIEMENS_3.0 -1.9693380 45.40364
## 20114            Philips3T -0.6518568 69.67572
## 20115            Philips3T  0.1625183 74.63609
## 20116            Philips3T  0.1583078 75.29285
## 20427            Philips3T  0.1786114 75.55633
## 20428            Philips3T  1.9573671 78.61181</code></pre>
<pre class="r"><code>## remove those people
raw_nodup &lt;- raw[raw$Phenotype_ID %!in% dupAge$Phenotype_ID,]

## extract baseline data
tmp &lt;- raw_nodup %&gt;%
  select(Phenotype_ID,PHC_Age_T1) %&gt;%
  group_by(Phenotype_ID) %&gt;%
  arrange(PHC_Age_T1) %&gt;%
  mutate(last = ifelse(row_number() == last(row_number()), T, F)) %&gt;%
  mutate(first = ifelse(row_number() == first(row_number()), T, F)) %&gt;%
  ungroup()

tmp &lt;- merge(tmp,raw_nodup, by=c(&quot;Phenotype_ID&quot;,&quot;PHC_Age_T1&quot;))

## get number of individuals with only one observation
ppID_oneObs &lt;- tmp[tmp$last == T &amp;  tmp$first == T,]
table(ppID_oneObs$Study)</code></pre>
<pre><code>## 
##       ADNI       NACC ROSMAPMARS     WHICAP       WRAP 
##        551       3796        462        678        205</code></pre>
<pre class="r"><code>baseline &lt;- tmp[tmp$first==T,]
paste0(&quot;Baseline count: &quot;, nrow(baseline))</code></pre>
<pre><code>## [1] &quot;Baseline count: 10178&quot;</code></pre>
<pre class="r"><code>paste0(&quot;Number of individuals: &quot;, length(unique(baseline$Phenotype_ID)))</code></pre>
<pre><code>## [1] &quot;Number of individuals: 10178&quot;</code></pre>
</div>
<div id="dataset-summary" class="section level1">
<h1>Dataset Summary</h1>
<div id="table-1" class="section level2">
<h2>Table 1</h2>
<pre class="r"><code>table1(~ Sex + PHC_Age_T1 + SPARE_AD + SPARE_BA + Race + PHC_Education + Diagnosis | Study, data=baseline)</code></pre>
<div class="Rtable1"><table class="Rtable1">
<thead>
<tr>
<th class='rowlabel firstrow lastrow'></th>
<th class='firstrow lastrow'><span class='stratlabel'>ADNI<br><span class='stratn'>(N=2532)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>NACC<br><span class='stratn'>(N=4861)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>ROSMAPMARS<br><span class='stratn'>(N=1182)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>WHICAP<br><span class='stratn'>(N=991)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>WRAP<br><span class='stratn'>(N=612)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Overall<br><span class='stratn'>(N=10178)</span></span></th>
</tr>
</thead>
<tbody>
<tr>
<td class='rowlabel firstrow'>Sex</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Female</td>
<td>1221 (48.2%)</td>
<td>2858 (58.8%)</td>
<td>917 (77.6%)</td>
<td>603 (60.8%)</td>
<td>424 (69.3%)</td>
<td>6023 (59.2%)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Male</td>
<td class='lastrow'>1311 (51.8%)</td>
<td class='lastrow'>2003 (41.2%)</td>
<td class='lastrow'>265 (22.4%)</td>
<td class='lastrow'>388 (39.2%)</td>
<td class='lastrow'>188 (30.7%)</td>
<td class='lastrow'>4155 (40.8%)</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Age_T1</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>72.9 (7.42)</td>
<td>71.1 (10.6)</td>
<td>79.5 (7.24)</td>
<td>75.5 (6.44)</td>
<td>62.5 (7.39)</td>
<td>72.4 (9.71)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>72.9 [50.5, 91.5]</td>
<td class='lastrow'>71.8 [21.5, 100]</td>
<td class='lastrow'>79.8 [58.8, 100]</td>
<td class='lastrow'>74.4 [62.5, 98.2]</td>
<td class='lastrow'>62.8 [43.2, 83.8]</td>
<td class='lastrow'>72.8 [21.5, 100]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_AD</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>-0.557 (1.33)</td>
<td>-0.725 (1.41)</td>
<td>-0.910 (1.01)</td>
<td>-1.19 (0.817)</td>
<td>-1.60 (0.790)</td>
<td>-0.803 (1.30)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>-0.790 [-3.79, 5.27]</td>
<td class='lastrow'>-1.02 [-4.00, 5.22]</td>
<td class='lastrow'>-1.01 [-4.45, 3.18]</td>
<td class='lastrow'>-1.27 [-3.72, 2.33]</td>
<td class='lastrow'>-1.60 [-4.60, 1.29]</td>
<td class='lastrow'>-1.05 [-4.60, 5.27]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_BA</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>62.2 (11.8)</td>
<td>60.1 (14.5)</td>
<td>65.8 (11.5)</td>
<td>60.7 (10.2)</td>
<td>47.9 (10.0)</td>
<td>60.6 (13.5)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>61.8 [27.4, 104]</td>
<td class='lastrow'>60.3 [-3.56, 109]</td>
<td class='lastrow'>65.8 [32.8, 105]</td>
<td class='lastrow'>59.9 [28.4, 99.3]</td>
<td class='lastrow'>47.5 [19.7, 81.5]</td>
<td class='lastrow'>60.5 [-3.56, 109]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>Race</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>American Indian or Alaska Native</td>
<td>4 (0.2%)</td>
<td>41 (0.8%)</td>
<td>3 (0.3%)</td>
<td>0 (0%)</td>
<td>6 (1.0%)</td>
<td>54 (0.5%)</td>
</tr>
<tr>
<td class='rowlabel'>Asian</td>
<td>60 (2.4%)</td>
<td>80 (1.6%)</td>
<td>5 (0.4%)</td>
<td>0 (0%)</td>
<td>1 (0.2%)</td>
<td>146 (1.4%)</td>
</tr>
<tr>
<td class='rowlabel'>Black or African American</td>
<td>198 (7.8%)</td>
<td>609 (12.5%)</td>
<td>271 (22.9%)</td>
<td>335 (33.8%)</td>
<td>26 (4.2%)</td>
<td>1439 (14.1%)</td>
</tr>
<tr>
<td class='rowlabel'>Native Hawaiian or Other Pacific Islander</td>
<td>2 (0.1%)</td>
<td>2 (0.0%)</td>
<td>1 (0.1%)</td>
<td>0 (0%)</td>
<td>0 (0%)</td>
<td>5 (0.0%)</td>
</tr>
<tr>
<td class='rowlabel'>Other, Unknown, or More than one race</td>
<td>48 (1.9%)</td>
<td>44 (0.9%)</td>
<td>8 (0.7%)</td>
<td>412 (41.6%)</td>
<td>3 (0.5%)</td>
<td>515 (5.1%)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>White</td>
<td class='lastrow'>2220 (87.7%)</td>
<td class='lastrow'>4085 (84.0%)</td>
<td class='lastrow'>894 (75.6%)</td>
<td class='lastrow'>244 (24.6%)</td>
<td class='lastrow'>576 (94.1%)</td>
<td class='lastrow'>8019 (78.8%)</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Education</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>16.1 (2.71)</td>
<td>15.5 (3.21)</td>
<td>15.8 (3.32)</td>
<td>12.2 (4.69)</td>
<td>16.4 (2.80)</td>
<td>15.4 (3.44)</td>
</tr>
<tr>
<td class='rowlabel'>Median [Min, Max]</td>
<td>16.0 [4.00, 20.0]</td>
<td>16.0 [0, 25.0]</td>
<td>16.0 [3.00, 30.0]</td>
<td>12.0 [0, 20.0]</td>
<td>16.0 [10.0, 29.0]</td>
<td>16.0 [0, 30.0]</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Missing</td>
<td class='lastrow'>0 (0%)</td>
<td class='lastrow'>9 (0.2%)</td>
<td class='lastrow'>0 (0%)</td>
<td class='lastrow'>3 (0.3%)</td>
<td class='lastrow'>1 (0.2%)</td>
<td class='lastrow'>13 (0.1%)</td>
</tr>
<tr>
<td class='rowlabel firstrow'>Diagnosis</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Alzheimer's Dementia</td>
<td>425 (16.8%)</td>
<td>834 (17.2%)</td>
<td>14 (1.2%)</td>
<td>46 (4.6%)</td>
<td>3 (0.5%)</td>
<td>1322 (13.0%)</td>
</tr>
<tr>
<td class='rowlabel'>Mild Cognitive Impairment</td>
<td>1140 (45.0%)</td>
<td>1139 (23.4%)</td>
<td>222 (18.8%)</td>
<td>174 (17.6%)</td>
<td>21 (3.4%)</td>
<td>2696 (26.5%)</td>
</tr>
<tr>
<td class='rowlabel'>No Cognitive Impairment</td>
<td>939 (37.1%)</td>
<td>2587 (53.2%)</td>
<td>941 (79.6%)</td>
<td>734 (74.1%)</td>
<td>586 (95.8%)</td>
<td>5787 (56.9%)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Missing</td>
<td class='lastrow'>28 (1.1%)</td>
<td class='lastrow'>301 (6.2%)</td>
<td class='lastrow'>5 (0.4%)</td>
<td class='lastrow'>37 (3.7%)</td>
<td class='lastrow'>2 (0.3%)</td>
<td class='lastrow'>373 (3.7%)</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="histogram-of-spare-ad" class="section level2">
<h2>Histogram of SPARE-AD</h2>
<p><img src="descriptive_analysis_files/figure-html/unnamed-chunk-10-1.png" width="672" /></p>
</div>
<div id="boxplot-of-spare-ad-by-diagnosis" class="section level2">
<h2>Boxplot of SPARE-AD by Diagnosis</h2>
<pre class="r"><code>p &lt;- ggboxplot(baseline[!is.na(baseline$PHC_Diagnosis),], x = &quot;PHC_Diagnosis&quot;, y = &quot;SPARE_AD&quot;,
          color = &quot;Diagnosis&quot;, palette = &quot;jco&quot;,
          add = &quot;jitter&quot;)
p + stat_compare_means(method = &quot;anova&quot;) #  Add p-value</code></pre>
<p><img src="descriptive_analysis_files/figure-html/unnamed-chunk-12-1.png" width="672" /></p>
</div>
</div>
<div id="prepare-genotype-data" class="section level1">
<h1>Prepare Genotype Data</h1>
<div id="adni" class="section level2">
<h2>ADNI</h2>
<p><strong>File path</strong>:
/nfs/DATA/ADNI/GWAS/Imputed/TOPMed/Cleaned</p>
<p><br></p>
<table>
<colgroup>
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Imputed PLINK File</th>
<th align="center">PCs</th>
<th align="center">Race</th>
<th align="center">Variants</th>
<th align="center">Participants</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">ADNI_NHW_imputed_final</td>
<td
align="center">ADNI_NHW_imputed_final_shorter_bimids_pruned.pca.evec</td>
<td align="center">NHW</td>
<td align="center">7,953,372</td>
<td align="center">1414</td>
</tr>
<tr class="even">
<td align="center">ADNI_NHB_imputed_final_nooutliers</td>
<td
align="center">ADNI_NHB_imputed_final_nooutliers_shorter_bimids_pruned.pca.evec</td>
<td align="center">NHB</td>
<td align="center">8,028,923</td>
<td align="center">58</td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Data Preparation Code at ACCRE and local</strong></p>
<pre class="r bggreen"><code>## mkdir
cd /data/h_vmac/zhanm32
mkdir mkdir SPARE_AD
cd SPARE_AD
mkdir DATA
mkdir CODE
mkdir OUTPUT
mkdir DATA/ADNI
mkdir DATA/NACC
mkdir DATA/ROSMAP

## copy geno data and PCs 
cp /nfs/DATA/ADNI/GWAS/Imputed/TOPMed/Cleaned/ADNI_NHW_imputed_final.* /data/h_vmac/zhanm32/SPARE_AD/DATA/ADNI

cp /nfs/DATA/ADNI/GWAS/Imputed/TOPMed/Cleaned/ADNI_NHW_imputed_final_shorter_bimids_pruned.pca.evec /data/h_vmac/zhanm32/SPARE_AD/DATA/ADNI

## download .fam and PC file to local (/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno) to generate final pheno_covs file
## code: /Users/mengnazhang/Documents/jinguameng.github.io/descriptive_analysis.Rmd

## remove the first line of PC file
cd /Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno

tail -n +2 ADNI_NHW_imputed_final_shorter_bimids_pruned.pca.evec &gt; ADNI_NHW_imputed_final_shorter_bimids_pruned.pca.nohead.evec</code></pre>
<p><strong>Prepare pheno_covs file</strong></p>
<pre class="r"><code>fam_ADNI &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/ADNI_NHW_imputed_final.fam&quot;,header = F)
pc_ADNI &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/ADNI_NHW_imputed_final_shorter_bimids_pruned.pca.nohead.evec&quot;,header = F)

## subset .fam file and rename the columns
fam_ADNI &lt;- fam_ADNI[,c(1:2)]
colnames(fam_ADNI) &lt;- c(&quot;FID&quot;,&quot;IID&quot;)
fam_ADNI$FIDIID &lt;- paste0(fam_ADNI$FID,&quot;:&quot;,fam_ADNI$IID)

## subset PCs file and rename the columns
pc_ADNI &lt;- pc_ADNI[,c(1:11)]
colnames(pc_ADNI) &lt;- c(&quot;FIDIID&quot;,paste0(&quot;PC&quot;,seq(1:10)))

## merge .fam file and PCs file
tmp &lt;- merge(fam_ADNI,pc_ADNI,by=&quot;FIDIID&quot;)

## merge with pheno data
baseline_ADNI &lt;- subset(baseline,baseline$Study == &quot;ADNI&quot;)
pheno_ADNI &lt;- merge(tmp,baseline_ADNI,by.x = &quot;IID&quot;,by.y = &quot;Phenotype_ID&quot;)
## reorder the columns
pheno_ADNI &lt;- pheno_ADNI[,c(3,1,2,4:32)]
## save the file
write.table(pheno_ADNI,&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Clean/ADNI_clean_pheno_covs.txt&quot;,col.names=T,row.names = F,quote = F,sep=&quot;\t&quot;)
paste0(&quot;Number of observations of final cleaned pheno_covs file for ADNI non-hispanic white: &quot;, nrow(pheno_ADNI))</code></pre>
<pre><code>## [1] &quot;Number of observations of final cleaned pheno_covs file for ADNI non-hispanic white: 1391&quot;</code></pre>
<p><br></p>
<div id="table-1-1" class="section level3">
<h3>Table 1</h3>
<div class="Rtable1"><table class="Rtable1">
<thead>
<tr>
<th class='rowlabel firstrow lastrow'></th>
<th class='firstrow lastrow'><span class='stratlabel'>Alzheimer's Dementia<br><span class='stratn'>(N=204)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Mild Cognitive Impairment<br><span class='stratn'>(N=659)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>No Cognitive Impairment<br><span class='stratn'>(N=528)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Overall<br><span class='stratn'>(N=1391)</span></span></th>
</tr>
</thead>
<tbody>
<tr>
<td class='rowlabel firstrow'>factor(Sex)</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Female</td>
<td>89 (43.6%)</td>
<td>254 (38.5%)</td>
<td>277 (52.5%)</td>
<td>620 (44.6%)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Male</td>
<td class='lastrow'>115 (56.4%)</td>
<td class='lastrow'>405 (61.5%)</td>
<td class='lastrow'>251 (47.5%)</td>
<td class='lastrow'>771 (55.4%)</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Age_T1</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>75.3 (8.14)</td>
<td>73.4 (7.57)</td>
<td>73.6 (5.99)</td>
<td>73.7 (7.13)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>76.0 [55.3, 90.9]</td>
<td class='lastrow'>73.7 [54.6, 91.5]</td>
<td class='lastrow'>73.0 [55.8, 90.3]</td>
<td class='lastrow'>73.6 [54.6, 91.5]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_AD</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>1.10 (1.12)</td>
<td>-0.366 (1.26)</td>
<td>-1.33 (0.800)</td>
<td>-0.518 (1.35)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>1.02 [-1.82, 5.27]</td>
<td class='lastrow'>-0.432 [-3.38, 2.98]</td>
<td class='lastrow'>-1.35 [-3.76, 2.30]</td>
<td class='lastrow'>-0.762 [-3.76, 5.27]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_BA</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>70.7 (10.5)</td>
<td>63.5 (11.8)</td>
<td>58.4 (9.74)</td>
<td>62.6 (11.6)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>70.6 [40.7, 95.3]</td>
<td class='lastrow'>64.1 [27.4, 102]</td>
<td class='lastrow'>58.1 [33.4, 92.3]</td>
<td class='lastrow'>62.4 [27.4, 102]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Education</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>15.1 (3.03)</td>
<td>16.0 (2.81)</td>
<td>16.6 (2.49)</td>
<td>16.1 (2.77)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>16.0 [6.00, 20.0]</td>
<td class='lastrow'>16.0 [6.00, 20.0]</td>
<td class='lastrow'>16.0 [6.00, 20.0]</td>
<td class='lastrow'>16.0 [6.00, 20.0]</td>
</tr>
</tbody>
</table>
</div>
<p><br></p>
</div>
<div id="box-plot" class="section level3">
<h3>Box plot</h3>
<p><img src="descriptive_analysis_files/figure-html/unnamed-chunk-16-1.png" width="672" /></p>
<p><br></p>
<p><strong>Upload ADNI_clean_pheno_covs.txt file to ACCRE:
/data/h_vmac/zhanm32/SPARE_AD/DATA/ADNI/</strong></p>
<p><br> <br></p>
</div>
</div>
<div id="nacc" class="section level2">
<h2>NACC</h2>
<p><strong>File path</strong>:
/nfs/DATA/NACC/GWAS/Imputed/TOPMed/Cleaned</p>
<p><br></p>
<table>
<colgroup>
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Imputed PLINK File</th>
<th align="center">PCs</th>
<th align="center">Race</th>
<th align="center">Variants</th>
<th align="center">Participants</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">NACC_NHW_imputed_final</td>
<td
align="center">NACC_NHW_imputed_final_shorter_bimids_pruned.pca.evec</td>
<td align="center">NHW</td>
<td align="center">7,432,354</td>
<td align="center">22572</td>
</tr>
<tr class="even">
<td align="center">NACC_NHB_imputed_final_nooutliers</td>
<td
align="center">NACC_NHB_imputed_final_nooutliers_shorter_bimids_pruned.pca.evec</td>
<td align="center">NHB</td>
<td align="center">6,007,563</td>
<td align="center">2919</td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Data Preparation Code at ACCRE and local</strong></p>
<pre class="r bggreen"><code>## copy geno data and PCs 
cp /nfs/DATA/NACC/GWAS/Imputed/TOPMed/Cleaned/NACC_NHW_imputed_final.* /data/h_vmac/zhanm32/SPARE_AD/DATA/NACC

cp /nfs/DATA/NACC/GWAS/Imputed/TOPMed/Cleaned/NACC_NHW_imputed_final_shorter_bimids_pruned.pca.evec /data/h_vmac/zhanm32/SPARE_AD/DATA/NACC

cp /nfs/DATA/NACC/GWAS/Imputed/TOPMed/Cleaned/NACC_NHB_imputed_final_nooutliers.* /data/h_vmac/zhanm32/SPARE_AD/DATA/NACC

cp /nfs/DATA/NACC/GWAS/Imputed/TOPMed/Cleaned/NACC_NHB_imputed_final_nooutliers_shorter_bimids_pruned.pca.evec /data/h_vmac/zhanm32/SPARE_AD/DATA/NACC

## download .fam and PC file to local (/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno) to generate final pheno_covs file
## code: /Users/mengnazhang/Documents/jinguameng.github.io/descriptive_analysis.Rmd

## remove the first line of PC file
cd /Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno

tail -n +2 NACC_NHB_imputed_final_nooutliers_shorter_bimids_pruned.pca.evec &gt; NACC_NHB_imputed_final_nooutliers_shorter_bimids_pruned.pca.nohead.evec

tail -n +2 NACC_NHW_imputed_final_shorter_bimids_pruned.pca.evec &gt; NACC_NHW_imputed_final_shorter_bimids_pruned.pca.nohead.evec</code></pre>
<p><strong>Prepare pheno_covs file</strong></p>
<pre class="r"><code>fam_NACC_NHW &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/NACC_NHW_imputed_final.fam&quot;,header = F)
pc_NACC_NHW &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/NACC_NHW_imputed_final_shorter_bimids_pruned.pca.nohead.evec&quot;,header = F)

fam_NACC_NHB &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/NACC_NHB_imputed_final_nooutliers.fam&quot;,header = F)
pc_NACC_NHB &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/NACC_NHB_imputed_final_nooutliers_shorter_bimids_pruned.pca.nohead.evec&quot;,header = F)

## subset .fam file and rename the columns
fam_NACC_NHW &lt;- fam_NACC_NHW[,c(1:2)]
colnames(fam_NACC_NHW) &lt;- c(&quot;FID&quot;,&quot;IID&quot;)
fam_NACC_NHW$FIDIID &lt;- paste0(fam_NACC_NHW$FID,&quot;:&quot;,fam_NACC_NHW$IID)

fam_NACC_NHB &lt;- fam_NACC_NHB[,c(1:2)]
colnames(fam_NACC_NHB) &lt;- c(&quot;FID&quot;,&quot;IID&quot;)
fam_NACC_NHB$FIDIID &lt;- paste0(fam_NACC_NHB$FID,&quot;:&quot;,fam_NACC_NHB$IID)

## subset PCs file and rename the columns
pc_NACC_NHW &lt;- pc_NACC_NHW[,c(1:11)]
colnames(pc_NACC_NHW) &lt;- c(&quot;FIDIID&quot;,paste0(&quot;PC&quot;,seq(1:10)))

pc_NACC_NHB &lt;- pc_NACC_NHB[,c(1:11)]
colnames(pc_NACC_NHB) &lt;- c(&quot;FIDIID&quot;,paste0(&quot;PC&quot;,seq(1:10)))

## merge .fam file and PCs file
tmp_NHW &lt;- merge(fam_NACC_NHW,pc_NACC_NHW,by=&quot;FIDIID&quot;)
tmp_NHB &lt;- merge(fam_NACC_NHB,pc_NACC_NHB,by=&quot;FIDIID&quot;)

## merge with pheno data
baseline_NACC &lt;- subset(baseline,baseline$Study == &quot;NACC&quot;)
pheno_NACC_NHW &lt;- merge(tmp_NHW,baseline_NACC,by.x = &quot;IID&quot;,by.y = &quot;Phenotype_ID&quot;)

pheno_NACC_NHB &lt;- merge(tmp_NHB,baseline_NACC,by.x = &quot;IID&quot;,by.y = &quot;Phenotype_ID&quot;)

## reorder the columns
pheno_NACC_NHW &lt;- pheno_NACC_NHW[,c(3,1,2,4:32)]

pheno_NACC_NHB &lt;- pheno_NACC_NHB[,c(3,1,2,4:32)]

## save the file
write.table(pheno_NACC_NHW,&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Clean/NACC_NHW_clean_pheno_covs.txt&quot;,col.names=T,row.names = F,quote = F,sep=&quot;\t&quot;)

write.table(pheno_NACC_NHB,&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Clean/NACC_NHB_clean_pheno_covs.txt&quot;,col.names=T,row.names = F,quote = F,sep=&quot;\t&quot;)

paste0(&quot;Number of observations of final cleaned pheno_covs file for NACC Non-Hispanic white: &quot;, nrow(pheno_NACC_NHW))</code></pre>
<pre><code>## [1] &quot;Number of observations of final cleaned pheno_covs file for NACC Non-Hispanic white: 2612&quot;</code></pre>
<pre class="r"><code>paste0(&quot;Number of observations of final cleaned pheno_covs file for NACC Non-Hispanic black: &quot;, nrow(pheno_NACC_NHB))</code></pre>
<pre><code>## [1] &quot;Number of observations of final cleaned pheno_covs file for NACC Non-Hispanic black: 372&quot;</code></pre>
<p><br></p>
<div id="table-1-of-nacc-non-hispanic-white" class="section level3">
<h3>Table 1 of NACC Non-Hispanic White</h3>
<pre><code>## 
##      Alzheimer&#39;s Dementia Mild Cognitive Impairment   No Cognitive Impairment 
##                       462                       612                      1386 
##                      &lt;NA&gt; 
##                       152</code></pre>
<div class="Rtable1"><table class="Rtable1">
<thead>
<tr>
<th class='rowlabel firstrow lastrow'></th>
<th class='firstrow lastrow'><span class='stratlabel'>Alzheimer's Dementia<br><span class='stratn'>(N=462)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Mild Cognitive Impairment<br><span class='stratn'>(N=612)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>No Cognitive Impairment<br><span class='stratn'>(N=1386)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Overall<br><span class='stratn'>(N=2460)</span></span></th>
</tr>
</thead>
<tbody>
<tr>
<td class='rowlabel firstrow'>factor(Sex)</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Female</td>
<td>208 (45.0%)</td>
<td>266 (43.5%)</td>
<td>879 (63.4%)</td>
<td>1353 (55.0%)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Male</td>
<td class='lastrow'>254 (55.0%)</td>
<td class='lastrow'>346 (56.5%)</td>
<td class='lastrow'>507 (36.6%)</td>
<td class='lastrow'>1107 (45.0%)</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Age_T1</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>74.3 (8.99)</td>
<td>74.2 (9.03)</td>
<td>69.9 (11.0)</td>
<td>71.8 (10.4)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>75.2 [43.5, 94.4]</td>
<td class='lastrow'>75.0 [27.7, 96.6]</td>
<td class='lastrow'>70.7 [21.9, 100]</td>
<td class='lastrow'>72.6 [21.9, 100]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_AD</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>1.19 (1.33)</td>
<td>-0.305 (1.24)</td>
<td>-1.44 (0.849)</td>
<td>-0.662 (1.46)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>1.14 [-2.69, 5.22]</td>
<td class='lastrow'>-0.338 [-3.71, 3.54]</td>
<td class='lastrow'>-1.51 [-3.89, 3.84]</td>
<td class='lastrow'>-0.959 [-3.89, 5.22]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_BA</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>72.4 (12.1)</td>
<td>65.2 (12.6)</td>
<td>55.1 (13.5)</td>
<td>60.9 (14.8)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>71.9 [17.0, 106]</td>
<td class='lastrow'>66.0 [19.3, 108]</td>
<td class='lastrow'>55.3 [10.5, 93.8]</td>
<td class='lastrow'>61.4 [10.5, 108]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Education</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>15.5 (2.98)</td>
<td>16.0 (2.77)</td>
<td>16.3 (2.61)</td>
<td>16.1 (2.74)</td>
</tr>
<tr>
<td class='rowlabel'>Median [Min, Max]</td>
<td>16.0 [6.00, 22.0]</td>
<td>16.0 [4.00, 23.0]</td>
<td>16.0 [0, 25.0]</td>
<td>16.0 [0, 25.0]</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Missing</td>
<td class='lastrow'>0 (0%)</td>
<td class='lastrow'>1 (0.2%)</td>
<td class='lastrow'>2 (0.1%)</td>
<td class='lastrow'>3 (0.1%)</td>
</tr>
</tbody>
</table>
</div>
<p><br></p>
</div>
<div id="box-plot-of-nacc-non-hispanic-white" class="section level3">
<h3>Box plot of NACC Non-Hispanic White</h3>
<p><img src="descriptive_analysis_files/figure-html/unnamed-chunk-21-1.png" width="672" /></p>
<p><br></p>
</div>
<div id="table-1-of-nacc-non-hispanic-black" class="section level3">
<h3>Table 1 of NACC Non-Hispanic Black</h3>
<pre><code>## 
##      Alzheimer&#39;s Dementia Mild Cognitive Impairment   No Cognitive Impairment 
##                        39                       101                       212 
##                      &lt;NA&gt; 
##                        20</code></pre>
<div class="Rtable1"><table class="Rtable1">
<thead>
<tr>
<th class='rowlabel firstrow lastrow'></th>
<th class='firstrow lastrow'><span class='stratlabel'>Alzheimer's Dementia<br><span class='stratn'>(N=39)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Mild Cognitive Impairment<br><span class='stratn'>(N=101)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>No Cognitive Impairment<br><span class='stratn'>(N=212)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Overall<br><span class='stratn'>(N=352)</span></span></th>
</tr>
</thead>
<tbody>
<tr>
<td class='rowlabel firstrow'>factor(Sex)</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Female</td>
<td>25 (64.1%)</td>
<td>72 (71.3%)</td>
<td>169 (79.7%)</td>
<td>266 (75.6%)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Male</td>
<td class='lastrow'>14 (35.9%)</td>
<td class='lastrow'>29 (28.7%)</td>
<td class='lastrow'>43 (20.3%)</td>
<td class='lastrow'>86 (24.4%)</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Age_T1</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>77.7 (8.41)</td>
<td>74.6 (8.08)</td>
<td>72.1 (7.17)</td>
<td>73.4 (7.79)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>77.7 [61.4, 95.0]</td>
<td class='lastrow'>74.7 [59.7, 95.6]</td>
<td class='lastrow'>71.2 [56.7, 92.4]</td>
<td class='lastrow'>72.6 [56.7, 95.6]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_AD</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>0.526 (1.59)</td>
<td>-0.757 (0.972)</td>
<td>-1.33 (0.713)</td>
<td>-0.959 (1.09)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>0.203 [-3.19, 4.32]</td>
<td class='lastrow'>-0.836 [-3.36, 2.31]</td>
<td class='lastrow'>-1.31 [-3.08, 0.645]</td>
<td class='lastrow'>-1.11 [-3.36, 4.32]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_BA</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>68.8 (12.5)</td>
<td>61.8 (10.5)</td>
<td>56.7 (10.3)</td>
<td>59.5 (11.3)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>71.0 [37.5, 92.5]</td>
<td class='lastrow'>61.3 [35.6, 88.5]</td>
<td class='lastrow'>56.7 [20.9, 92.4]</td>
<td class='lastrow'>58.9 [20.9, 92.5]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Education</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>13.4 (2.93)</td>
<td>15.0 (2.76)</td>
<td>14.9 (2.80)</td>
<td>14.8 (2.84)</td>
</tr>
<tr>
<td class='rowlabel'>Median [Min, Max]</td>
<td>12.0 [8.00, 20.0]</td>
<td>16.0 [7.00, 22.0]</td>
<td>14.0 [5.00, 22.0]</td>
<td>14.0 [5.00, 22.0]</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Missing</td>
<td class='lastrow'>1 (2.6%)</td>
<td class='lastrow'>0 (0%)</td>
<td class='lastrow'>0 (0%)</td>
<td class='lastrow'>1 (0.3%)</td>
</tr>
</tbody>
</table>
</div>
<p><br></p>
</div>
<div id="box-plot-of-nacc-non-hispanic-black" class="section level3">
<h3>Box plot of NACC Non-Hispanic Black</h3>
<p><img src="descriptive_analysis_files/figure-html/unnamed-chunk-23-1.png" width="672" /></p>
<p><strong>Upload NACC_NHW_clean_pheno_covs.txt and
NACC_NHB_clean_pheno_covs.txt files to ACCRE:
/data/h_vmac/zhanm32/SPARE_AD/DATA/NACC/</strong></p>
<p><br> <br></p>
</div>
</div>
<div id="rosmap" class="section level2">
<h2>ROSMAP</h2>
<p><strong>File path</strong>:
/nfs/DATA/ROSMAP/GWAS/Imputed/TOPMed/Cleaned</p>
<p><br></p>
<table>
<colgroup>
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Imputed PLINK File</th>
<th align="center">PCs</th>
<th align="center">Race</th>
<th align="center">Variants</th>
<th align="center">Participants</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">ROSMAP_NHW_imputed_final</td>
<td
align="center">ROSMAP_NHW_imputed_final_shorter_bimids_pruned.pca.evec</td>
<td align="center">NHW</td>
<td align="center">7,624,508</td>
<td align="center">2120</td>
</tr>
<tr class="even">
<td align="center">ROSMAP_NHB_final_updatedIDs_nooutliers</td>
<td
align="center">ROSMAP_NHB_final_updatedIDs_nooutliers_shorter_bimids_pruned.pca.evec</td>
<td align="center">NHB</td>
<td align="center">5,120,071</td>
<td align="center">532</td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>Data Preparation Code at ACCRE and local</strong></p>
<pre class="r bggreen"><code>## copy geno data and PCs 
cp /nfs/DATA/ROSMAP/GWAS/Imputed/TOPMed/Cleaned/ROSMAP_NHW_imputed_final.* /data/h_vmac/zhanm32/SPARE_AD/DATA/ROSMAP

cp /nfs/DATA/ROSMAP/GWAS/Imputed/TOPMed/Cleaned/ROSMAP_NHW_imputed_final_shorter_bimids_pruned.pca.evec /data/h_vmac/zhanm32/SPARE_AD/DATA/ROSMAP

cp /nfs/DATA/ROSMAP/GWAS/Imputed/TOPMed/Cleaned/ROSMAP_NHB_final_updatedIDs_nooutliers.* /data/h_vmac/zhanm32/SPARE_AD/DATA/ROSMAP

cp /nfs/DATA/ROSMAP/GWAS/Imputed/TOPMed/Cleaned/ROSMAP_NHB_final_updatedIDs_nooutliers_shorter_bimids_pruned.pca.evec /data/h_vmac/zhanm32/SPARE_AD/DATA/ROSMAP

## download .fam and PC file to local (/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno) to generate final pheno_covs file
## code: /Users/mengnazhang/Documents/jinguameng.github.io/descriptive_analysis.Rmd

## remove the first line of PC file
cd /Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno

tail -n +2 ROSMAP_NHB_final_updatedIDs_nooutliers_shorter_bimids_pruned.pca.evec &gt; ROSMAP_NHB_final_updatedIDs_nooutliers_shorter_bimids_pruned.pca.nohead.evec

tail -n +2 ROSMAP_NHW_imputed_final_shorter_bimids_pruned.pca.evec &gt; ROSMAP_NHW_imputed_final_shorter_bimids_pruned.pca.nohead.evec</code></pre>
<p><strong>Prepare pheno_covs file</strong></p>
<pre class="r"><code>fam_ROSMAP_NHW &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/ROSMAP_NHW_imputed_final.fam&quot;,header = F)
pc_ROSMAP_NHW &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/ROSMAP_NHW_imputed_final_shorter_bimids_pruned.pca.nohead.evec&quot;,header = F)


fam_ROSMAP_NHB &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/ROSMAP_NHB_final_updatedIDs_nooutliers.fam&quot;,header = F)
pc_ROSMAP_NHB &lt;- read.table(&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Raw/geno/ROSMAP_NHB_final_updatedIDs_nooutliers_shorter_bimids_pruned.pca.nohead.evec&quot;,header = F)

## subset .fam file and rename the columns
fam_ROSMAP_NHW &lt;- fam_ROSMAP_NHW[,c(1:2)]
colnames(fam_ROSMAP_NHW) &lt;- c(&quot;FID&quot;,&quot;IID&quot;)
fam_ROSMAP_NHW$FIDIID &lt;- paste0(fam_ROSMAP_NHW$FID,&quot;:&quot;,fam_ROSMAP_NHW$IID)

fam_ROSMAP_NHB &lt;- fam_ROSMAP_NHB[,c(1:2)]
colnames(fam_ROSMAP_NHB) &lt;- c(&quot;FID&quot;,&quot;IID&quot;)
fam_ROSMAP_NHB$FIDIID &lt;- paste0(fam_ROSMAP_NHB$FID,&quot;:&quot;,fam_ROSMAP_NHB$IID)

## subset PCs file and rename the columns
pc_ROSMAP_NHW &lt;- pc_ROSMAP_NHW[,c(1:11)]
colnames(pc_ROSMAP_NHW) &lt;- c(&quot;FIDIID&quot;,paste0(&quot;PC&quot;,seq(1:10)))

pc_ROSMAP_NHB &lt;- pc_ROSMAP_NHB[,c(1:11)]
colnames(pc_ROSMAP_NHB) &lt;- c(&quot;FIDIID&quot;,paste0(&quot;PC&quot;,seq(1:10)))

## merge .fam file and PCs file
tmp_NHW &lt;- merge(fam_ROSMAP_NHW,pc_ROSMAP_NHW,by=&quot;FIDIID&quot;) ## 532
tmp_NHB &lt;- merge(fam_ROSMAP_NHB,pc_ROSMAP_NHB,by=&quot;FIDIID&quot;) ## 2120

## merge with pheno data
baseline_ROSMAP &lt;- subset(baseline,baseline$Study == &quot;ROSMAPMARS&quot;)
pheno_ROSMAP_NHW &lt;- merge(tmp_NHW,baseline_ROSMAP,by.x = &quot;IID&quot;,by.y = &quot;Phenotype_ID&quot;)

pheno_ROSMAP_NHB &lt;- merge(tmp_NHB,baseline_ROSMAP,by.x = &quot;IID&quot;,by.y = &quot;Phenotype_ID&quot;)

## reorder the columns
pheno_ROSMAP_NHW &lt;- pheno_ROSMAP_NHW[,c(3,1,2,4:32)]

pheno_ROSMAP_NHB &lt;- pheno_ROSMAP_NHB[,c(3,1,2,4:32)]

## save the file
write.table(pheno_ROSMAP_NHW,&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Clean/ROSMAP_NHW_clean_pheno_covs.txt&quot;,col.names=T,row.names = F,quote = F,sep=&quot;\t&quot;)

write.table(pheno_ROSMAP_NHB,&quot;/Users/mengnazhang/Desktop/SPARE-AD/Data/Clean/ROSMAP_NHB_clean_pheno_covs.txt&quot;,col.names=T,row.names = F,quote = F,sep=&quot;\t&quot;)

paste0(&quot;Number of observations of final cleaned pheno_covs file for ROSMAP Non-Hispanic white: &quot;, nrow(pheno_ROSMAP_NHW))</code></pre>
<pre><code>## [1] &quot;Number of observations of final cleaned pheno_covs file for ROSMAP Non-Hispanic white: 505&quot;</code></pre>
<pre class="r"><code>paste0(&quot;Number of observations of final cleaned pheno_covs file for ROSMAP Non-Hispanic black: &quot;, nrow(pheno_ROSMAP_NHB))</code></pre>
<pre><code>## [1] &quot;Number of observations of final cleaned pheno_covs file for ROSMAP Non-Hispanic black: 136&quot;</code></pre>
<p><br></p>
<div id="table-1-of-rosmap-non-hispanic-white" class="section level3">
<h3>Table 1 of ROSMAP Non-Hispanic White</h3>
<pre><code>## 
##      Alzheimer&#39;s Dementia Mild Cognitive Impairment   No Cognitive Impairment 
##                         5                        95                       405</code></pre>
<div class="Rtable1"><table class="Rtable1">
<thead>
<tr>
<th class='rowlabel firstrow lastrow'></th>
<th class='firstrow lastrow'><span class='stratlabel'>Alzheimer's Dementia<br><span class='stratn'>(N=5)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Mild Cognitive Impairment<br><span class='stratn'>(N=95)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>No Cognitive Impairment<br><span class='stratn'>(N=405)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Overall<br><span class='stratn'>(N=505)</span></span></th>
</tr>
</thead>
<tbody>
<tr>
<td class='rowlabel firstrow'>factor(Sex)</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Female</td>
<td>4 (80.0%)</td>
<td>73 (76.8%)</td>
<td>312 (77.0%)</td>
<td>389 (77.0%)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Male</td>
<td class='lastrow'>1 (20.0%)</td>
<td class='lastrow'>22 (23.2%)</td>
<td class='lastrow'>93 (23.0%)</td>
<td class='lastrow'>116 (23.0%)</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Age_T1</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>87.1 (4.68)</td>
<td>84.4 (5.76)</td>
<td>80.1 (7.14)</td>
<td>81.0 (7.11)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>84.9 [82.7, 94.4]</td>
<td class='lastrow'>84.4 [71.6, 99.8]</td>
<td class='lastrow'>80.6 [58.8, 100]</td>
<td class='lastrow'>81.8 [58.8, 100]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_AD</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>1.39 (0.859)</td>
<td>-0.295 (1.14)</td>
<td>-0.971 (0.931)</td>
<td>-0.820 (1.03)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>1.33 [0.177, 2.58]</td>
<td class='lastrow'>-0.396 [-3.17, 3.01]</td>
<td class='lastrow'>-1.03 [-3.32, 2.10]</td>
<td class='lastrow'>-0.925 [-3.32, 3.01]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_BA</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>83.0 (5.90)</td>
<td>73.4 (10.5)</td>
<td>65.8 (11.4)</td>
<td>67.4 (11.7)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>83.5 [74.2, 89.9]</td>
<td class='lastrow'>73.6 [48.8, 105]</td>
<td class='lastrow'>65.6 [37.4, 104]</td>
<td class='lastrow'>67.6 [37.4, 105]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Education</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>14.0 (2.45)</td>
<td>15.3 (2.89)</td>
<td>16.0 (3.24)</td>
<td>15.8 (3.18)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>14.0 [12.0, 18.0]</td>
<td class='lastrow'>16.0 [8.00, 25.0]</td>
<td class='lastrow'>16.0 [6.00, 30.0]</td>
<td class='lastrow'>16.0 [6.00, 30.0]</td>
</tr>
</tbody>
</table>
</div>
<p><br></p>
</div>
<div id="table-1-of-rosmap-non-hispanic-black" class="section level3">
<h3>Table 1 of ROSMAP Non-Hispanic Black</h3>
<pre><code>## 
##      Alzheimer&#39;s Dementia Mild Cognitive Impairment   No Cognitive Impairment 
##                         1                        20                       115</code></pre>
<div class="Rtable1"><table class="Rtable1">
<thead>
<tr>
<th class='rowlabel firstrow lastrow'></th>
<th class='firstrow lastrow'><span class='stratlabel'>Alzheimer's Dementia<br><span class='stratn'>(N=1)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Mild Cognitive Impairment<br><span class='stratn'>(N=20)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>No Cognitive Impairment<br><span class='stratn'>(N=115)</span></span></th>
<th class='firstrow lastrow'><span class='stratlabel'>Overall<br><span class='stratn'>(N=136)</span></span></th>
</tr>
</thead>
<tbody>
<tr>
<td class='rowlabel firstrow'>factor(Sex)</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Female</td>
<td>1 (100%)</td>
<td>15 (75.0%)</td>
<td>100 (87.0%)</td>
<td>116 (85.3%)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Male</td>
<td class='lastrow'>0 (0%)</td>
<td class='lastrow'>5 (25.0%)</td>
<td class='lastrow'>15 (13.0%)</td>
<td class='lastrow'>20 (14.7%)</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Age_T1</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>83.5 (NA)</td>
<td>78.7 (6.68)</td>
<td>77.0 (5.93)</td>
<td>77.3 (6.05)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>83.5 [83.5, 83.5]</td>
<td class='lastrow'>78.8 [63.9, 89.2]</td>
<td class='lastrow'>77.0 [64.4, 92.9]</td>
<td class='lastrow'>77.5 [63.9, 92.9]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_AD</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>2.69 (NA)</td>
<td>0.0950 (1.09)</td>
<td>-1.12 (0.742)</td>
<td>-0.910 (0.956)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>2.69 [2.69, 2.69]</td>
<td class='lastrow'>-0.0302 [-1.77, 3.18]</td>
<td class='lastrow'>-1.11 [-3.25, 0.583]</td>
<td class='lastrow'>-1.01 [-3.25, 3.18]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>SPARE_BA</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>78.4 (NA)</td>
<td>68.4 (7.75)</td>
<td>62.4 (9.91)</td>
<td>63.4 (9.88)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>78.4 [78.4, 78.4]</td>
<td class='lastrow'>69.9 [46.5, 82.2]</td>
<td class='lastrow'>62.9 [34.8, 90.3]</td>
<td class='lastrow'>63.6 [34.8, 90.3]</td>
</tr>
<tr>
<td class='rowlabel firstrow'>PHC_Education</td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
<td class='firstrow'></td>
</tr>
<tr>
<td class='rowlabel'>Mean (SD)</td>
<td>16.0 (NA)</td>
<td>15.3 (3.08)</td>
<td>15.3 (3.37)</td>
<td>15.3 (3.31)</td>
</tr>
<tr>
<td class='rowlabel lastrow'>Median [Min, Max]</td>
<td class='lastrow'>16.0 [16.0, 16.0]</td>
<td class='lastrow'>15.5 [8.00, 20.0]</td>
<td class='lastrow'>15.0 [8.00, 30.0]</td>
<td class='lastrow'>15.0 [8.00, 30.0]</td>
</tr>
</tbody>
</table>
</div>
<p><br></p>
<p><strong>Upload ROSMAP_NHW_clean_pheno_covs.txt and
ROSMAP_NHB_clean_pheno_covs.txt files to ACCRE:
/data/h_vmac/zhanm32/SPARE_AD/DATA/ROSMAP/</strong></p>
<p><br> <br></p>
</div>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
